snuba/datasets/storages/factory.py

```diff
@@ -69,13 +72,16 @@
     **(DEV_WRITABLE_STORAGES if settings.ENABLE_DEV_FEATURES else {}),
 }
 
-DEV_NON_WRITABLE_STORAGES: Mapping[StorageKey, ReadableTableStorage] = {
+DEV_NON_WRITABLE_STORAGES: Mapping[StorageKey, ReadableTableStorage] = {}
+
+METRICS_NON_WRITABLE_STORAGES: Mapping[StorageKey, ReadableTableStorage] = {
```

Comment:
I don't think you need to declare metrics storages in a separate variable anymore. You can just add them to the list at line 88.